storm-core/src/jvm/org/apache/storm/trident/planner/Node.java

```diff
@@ -25,7 +26,7 @@
 import org.apache.commons.lang.builder.ToStringStyle;
 
 
-public class Node implements Serializable {
+public class Node extends DefaultResourceDeclarer implements Serializable {
```

Comment:
And if DefaultResoureDeclarer is Generic this would change to.

```
public class Node extends DefaultResourceDeclarer<Node> implements Serializable {
```